similarity/README.md

```diff
@@ -2,7 +2,59 @@
 
 This module specifies a simple API for NLP components that compare
 objects -- especially Strings -- and return a score indicating how
-similar they are.  It is used in our WordNet-, Named Entity-, embedding-, 
-and paraphrase-based similarity code to simplify integration of 
+similar they are.  It is used in our WordNet-, Named Entity-, embedding-,
+and paraphrase-based similarity code to simplify integration of
 different similarity resources.
- 
\ No newline at end of file
+
+## Download Resource File
+
+To download resource file including: Word2vec, Glove, ESA, simply run the main function in Datastore class in `edu.illinois.cs.cogcomp.sim` package. It will download those file from Cogcomp server to `src/main/resources/1.5` in your local machine. You can also comment out the resource that you don't want or use your own file. Notice: "paragram" is already included in the `src/main/resources/`.
+
```

Comment:
I don't see why this makes the user's life is harder. Let's say the user has decided to `phrase2vec` dataset (say by setting it in the configurator or constructor options). The program would call datastore: 
```
File f = dsNoCredentials.getFile("org.cogcomp.wordembedding", "phrase2vec.txt", 1.5);
```
and we would use `f` for the rest of the run can be done with this file. Note that you don't need to copy anything to "src/main/resources/", because the file downloaded from datastore `f` already contains the location of the file. You can read it directly.